ddp-workspace/projects/ddp-dsm-ui/src/app/participant-page/participant-page.component.ts

```diff
@@ -190,7 +190,9 @@ export class ParticipantPageComponent implements OnInit, OnDestroy, AfterViewChe
 
     this.displayActivityOrder();
     this.addMedicalProviderInformation();
-    this.getMercuryEligibleSamples();
+    if(this.role.allowedToDoOrderSequencing()) {
+      this.getMercuryEligibleSamples();
+    }
     this.canSequence = this.canHaveSequencing(this.participant);
```

Comment:
should we also move that to the new if? we do not need to check for whatever this is checking if the person do not have the permissions

Reply:
Done!

Reply:
moved  `this.canSequence = this.canHaveSequencing(this.participant)` into if block